retroliste.com.domain-verification.json

```diff
@@ -0,0 +1,21 @@
+{
+  "providerId":"retroliste.com",
+  "providerName":"RetroListe",
+  "serviceId":"domain-verification",
+  "serviceName":"Domain Verification",
+  "sharedServiceName": false,
+  "version": 1,
+  "logoUrl":"https://retroliste.com/logo.gif",
+  "description":"Enables a domain to work with RetroListe",
+  "variableDescription":"Unique verification code provided for purpose of verification",
+  "multiInstance": false,
+  "records":[
+    {
+      "groupId": "verification",
+      "type": "TXT",
+      "host": "@",
+      "data": "%verifytxt%",
```

Comment:
This template, due to very open variable of TXT record can be considered insecure.
Please apply measures as per https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#security-considerations